pkg/server/biz/scm/bitbucket/server/server.go

```diff
@@ -334,19 +334,7 @@ func (b *BitbucketServer) CreateWebhook(repo string, webhook *scm.Webhook) error
 		for _, e := range webhook.Events {
 			switch e {
 			case scm.PullRequestEventType:
-				version, err := GetBitbucketVersion(b.v1Client.client, b.v1Client.baseURL)
-				if err != nil {
-					return err
-				}
-				isSupportPrModified, err := IsHigherVersion(version, SupportPrModifiedEvent)
```

Comment:
Why delete this?

Reply:
because we don't need to check the version now.

Reply:
I think we only want to check the version if we want to know if `PrModifiedEvent` is supported, but now `PrModifiedEvent` is not used anymore, so I delete these lines.